billing/data/egress.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from '@ucanto/server'
+import { DecodeFailure, EncodeFailure, Schema } from './lib.js'
+
+export const egressSchema = Schema.struct({
```

Comment:
issue (blocking): this file (and a few others) use 4 space indents, but we usually use 2 spaces - could you update across the board?

Reply:
I can format and include the egress tracking-related files in this PR, but for the unrelated files, I’d prefer to create a separate PR.